### DISCUSSION TITLE
Specify local file for custom filter list?

### DISCUSSION THREAD

**Discussion by tdisk** [uBlockOrigin/uBlock-issues#1754] [Q&A] [answered]
Hi all.  Github newb here.

subj: uBlock - Filter Lists | Custom | Import

Does anyone know if you can specify a *local* filter list here, e.g. using a "FILE:" type URL?
I was hoping to use ublockorigin to block a specific set of servers that I could specify n easily modify.
If not, then is there another option in ublockorigin that might accomplish the same results?

Thanks in advance.

**Comment by gorhill** [comment, marked as answer]
You can use `file:` URLs if your browser allows extensions to use `file:` URLs. In Chromium, it's opt-in, you have to enable _"Allow access to file URLs"_ in extension details page. In Firefox, it's not possible.

**Reply by tdisk** [reply]
Thanks for the reply!  Yes, I'm using firefox, so I guess I'm outta luck. [shrug]
